subiquitycore/models/network.py

```diff
@@ -360,6 +369,7 @@ def get_menu(self):
     # --- Model Methods ----
     def probe_network(self):
         log.debug('model calling prober.get_network()')
+        self.prober.probe()
```

Comment:
In case the user has connected an interface since that happened. Ogra has filed https://bugs.launchpad.net/ubuntu/+source/subiquity/+bug/1624322 since I did this, so maybe I should dust of my "live updates of networking UI" code.